src/backend/api/geese-features/del.js

```diff
@@ -0,0 +1,17 @@
+import db from '../../db';
+
+export default (req, res) => {
+  db.geeseFeatures
+    .deleteFeature(req.params.id)
+    .then((response) => {
+      if (response === 1) {
```

Comment:
We just check `if (response)` exists here, no need to check its value (we do the same for other DELETE API routes)